python/grass/script/core.py

```diff
@@ -66,6 +100,13 @@ def __init__(self, args, **kwargs):
         if cmd is None:
             raise OSError(_("Cannot find the executable {0}").format(args[0]))
         args = [cmd] + args[1:]
+
+        # Use text mode by default
+        self._implied_text = False
+        if "text" not in kwargs and "universal_newlines" not in kwargs:
+            kwargs["text"] = True
```

Comment:
Do we have a tool (module) or extension somewhere that has an option named `text` that would conflict with this? (Assuming Popen and the tool kwargs are still mixed together)

Reply:
We do, but this does not conflict because it is not mixed together anymore. From here, it goes directly to Popen parameters, the tools and its parameters are already a list of strings passed separately. That's different than run_command and family. The only issue is if you want to pass text=False to run_command - we don't support that (text is not in _popen_args in core.py which is used to make the distinction).

Reply:
So we leave like this or have to do something

Reply:
No action needed for text=True and potential conflict with tool parameters.